toolboxTreppe/toolboxTreppe.ino

```diff
@@ -93,23 +93,22 @@ void animation1() {
       ++m_x;
       xA = 5 - m_x;
       xB = 5 + m_x;
-      if (m_x < 0) {
-        m_x = 0;
-      }else if (m_x > 5) {
+      if (m_x > 5) {
         m_x = 0;
         ++m_y;
         //hue++;
         fadeToBlackBy(leds, NUM_LEDS, 20);
-        if (m_y > kMatrixHeight-1 || m_y < 0) {
+        if (m_y > kMatrixHeight-1) {
           m_y = 0;
         }
       }
     }
     if (currentMillis - onewayMillis > onewayTime) {
       onewayMillis = currentMillis;
-      --gegenstrecke;
-      if (gegenstrecke < 0 || gegenstrecke > kMatrixHeight-1) {
+      if (gegenstrecke == 0) {
```

Comment:
das scheint mir falsch -> !=

Reply:
wieso der aktuelle Check ist ist Gegenstrecke < 0 oder > kMatrixheight-1
es kann aber nie größer als kmatrixHeight-1 sein da nur --gegenstrecke gemacht wird
also ist der check nach oben hin schonmal sinnlos

der check nach unten wurde ersetzt durch ==0 da immer -1 gemacht wird, das jetzt aber erst nach diesem check gemacht wird.
Dh in dem Durchlauf in dem es bisher auf -1 gestellt wurde und auf kmatrixHeight -1 wird es jetzt einfach direkt auf kmatrixheight-1 gesetzt oder sehe ich das falsch?